quickstart/aws/tutorial-eks.md

```diff
@@ -11,22 +11,45 @@ In this tutorial, we'll launch a new Managed Kubernetes cluster in Elastic Conta
 
 1.  In a new folder `eks-hello-world`, create an empty project with `pulumi new`.
 
+    This will create a base Pulumi program in TypeScript, and is great
+    recommendation to begin your journey.
+
     ```bash
     $ pulumi new aws-typescript --dir eks-hello-world
     ```
 
-    Enter in a project name, description, stack name, and AWS region to
-    deploy into. We recommend using `us-west-2` to host your EKS cluster.
+    * Enter in a Pulumi project name, and description to detail what this
+      Pulumi program does
+    * Enter in a name for the [Pulumi stack](https://pulumi.io/reference/stack.html), which is an instance of our Pulumi program, and is used to distinguish amongst different development phases and environments of your work streams.
+    * Select 'no' when prompted to 'perform this update,' as we'll be
```

Comment:
@justinvp Is the plan to stop prompting to do an update? This was really confusing for Joe and I still find it very jarring personally, since 9/10, I want "no".

Reply:
I also find it jarring. It's on my list of things to address in the CLI this milestone.